ansible/roles/postldap/defaults/main.yml

```diff
@@ -318,18 +335,18 @@ postldap__postfix__dependent_lookup_tables:
     connection: '{{ postldap__postfix_ldap_connection }}'
     search_base: '{{ postldap__ldap_people_dn|join(",") }}'
     query_filter: '(&
-                    (objectClass=inetOrgPerson)
+                    (objectClass=mailRecipient)
                     (|
-                      (mail=%s)
-                      (mailAlias=%s)
+                      (mailAddress=%s)
+                      (mailAlternateAddress=%s)
```

Comment:
This part:
```
(|
  (mailAddress=%s)
  (mailAlternateAddress=%s)
)
```
can be replaced with:
```
(mailAddress=%s)
```
because aliases are already handled by the ldap_virtual_alias_maps.cf lookup table.